data/web/js/site/mailbox.js

```diff
@@ -1329,7 +1328,7 @@ jQuery(function($){
           defaultContent: '',
           responsivePriority: 4,
           render: function (data, type) {
-            return 1==data?'<i class="bi bi-check-lg"></i>':(0==data?'<i class="bi bi-x-lg"></i>':2==data&&'&#8212;');
+            return 1==data?'<i class="bi bi-check-lg"><span class="sorting-value">1</span></i>':(0==data?'<i class="bi bi-x-lg"><span class="sorting-value">0</span></i>':2==data&&'&#8212;');
```

Comment:
Thanks for the PR. Works so far, but could you check the mailbox table again?
There are 3 states (0, 1, 2). Please add the sorting value also for the last state.

Reply:
It's already sort able as it has the value [`&#8212;`](https://www.codetable.net/decimal/8212) which is shown as a dash. But i decided to not hide it as there was no Icon representing this state.